internal/description/description.go

```diff
@@ -91,7 +91,10 @@ const (
 	StartOnlineArchive       = "Start a paused online archive from a cluster."
 	UpdateOnlineArchive      = "Update an online archive for a cluster."
 	Search                   = "Manage search for your cluster."
+	Indexes                  = "Manage search indexes for your cluster."
 	ListSearchIndexes        = "List search indexes for a cluster."
+	CreateSearchIndexes      = "Create a search indexes for a cluster."
```

Comment:
"Create a search indexes for a cluster." -> "Create a search index for a cluster."
(and same below for delete)